src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -1179,6 +1220,11 @@ FixedwingAttitudeControl::task_main()
 				_actuators.control[actuator_controls_s::INDEX_THROTTLE] = _manual.z;
 			}
 
+			// Add feed-forward from roll control output to yaw control output
+			// This can be used to counteract the adverse yaw effect when rolling the plane
+			_actuators.control[actuator_controls_s::INDEX_YAW] += _parameters.roll_to_yaw_ff * math::constrain(
```

Comment:
I thought it's good here next to trims and such. If the mixers were easily configurable I would even put it there. (However, discussing this with some acrobatics pilots at the moment about whether this mixing should be mode dependent. So far I got various answers..)